package.json

```diff
@@ -10,7 +10,7 @@
     "libp2p-ipfs-nodejs": "libp2p-ipfs-browser",
     "./src/core/default-repo.js": "./src/core/default-repo-browser.js",
     "./src/core/components/init-assets.js": false,
-    "./test/utils/temp-repo.js": "./test/utils/temp-repo-browser.js",
+    "./test/utils/create-repo-node.js": "./test/utils/create-repo-browser.js",
```

Comment:
it could, I just standardize it because we always tag `node` and `browser` everywhere to distinguish the two, if distinction needs to happen